docs/_includes/lightbox-dialog.html

```diff
@@ -117,7 +114,6 @@ <h3 id="lightbox-scrolling">Expanded Lightbox</h3>
             <button class="btn btn--primary dialog-button" data-makeup-for="lightbox-dialog-expanded" type="button">Open Scrolling Lightbox Dialog</button>
```

Comment:
This text seems to match the previous dialog button. Can we change the name here to "Open Expanded Lightbox Dialog?" to match the specific dialog.

Reply:
I realize this was not caused by this PR, but we're cleaning things up, after all. Can we just make the change? It's a simple change, adds clarity and reduces confusion.